src/integration-test/java/fi/hsl/transitdata/tripupdate/application/ITTripUpdateProcessor.java

```diff
@@ -65,7 +65,7 @@ private void testValidCancellation(String routeName, String expectedRouteName, i
             @Override
             public void testImpl(TestPipeline.TestContext context) throws Exception {
                 final long ts = System.currentTimeMillis();
-                InternalMessages.TripCancellation cancellation = MockDataUtils.mockTripCancellation(routeName, joreDir, dateTime);
+                InternalMessages.TripCancellation cancellation = MockDataUtils.mockTripCancellation(routeName, joreDir, dateTime, Long.toString(dvjId));
```

Comment:
Would it make sense to have a `mockTripCancellation()` which would take `long dvjId` as a parameter?

Reply:
yes, good catch. I'll make everything consistent and use dvjId as long